src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XAttribute.cs

```diff
@@ -67,7 +69,7 @@ public XAttribute(XName name, object value)
         /// <exception cref="ArgumentNullException">
         /// Thrown if the specified <see cref="XAttribute"/> is null.
         /// </exception>
-        public XAttribute(XAttribute other)
+        public XAttribute([NotNull] XAttribute other)
```

Comment:
why is `[NotNull]` needed here?

Reply:
It was related to XElement.cs line 116; but taking a second look it doesn't really make sense to add NotNull here, I can use `!` on the call-site instead.